conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,345 @@
+          max: 1500m                  # 1500m * N where N = Num of "ios" Containers
+      - name: confd
+          memory:
+          min: 128Mi
+          max: 512Mi
+        cpu:
+          min: 250m
+          max: 500m
+    num_clients: 2
+    clients:                          # Motr Client Settings                 
+      - name: s3                     # s3|s3|motr_client (Motr Clients)
+        num_instances: 1              # Client instances per node (pod)
+        endpoints:
+        - tcp://server1-node1:21001   # Format: <protocol>://<hostname_ip>:21001 
+        - tcp://server2-node1:21001   # Num endpoints will be same as num of client instances
+                                      # Port range would be 21001 - 21001+num_instances
+                                      # Same endpoint will be used for s3 admin CLI
+      - name: motr_client             
+        num_instances: 0              # Optional client. Only for Dev.         
+        endpoints:
+        - tcp://client-node1:21201
+        - tcp://client-node2:21201
+
+cluster:
+  id: 0007ec45379e36d9fa089a3d615c32a3
+  name: cortx-cluster
+  storage_set_count: 1
+  storage_set:
+  - name: storage-set-1
+    durability:
+      dix:
+        data: '1'
+        parity: '0'
+        spare: '0'
+      sns:
+        data: '1'
+        parity: '0'
+        spare: '0'
+    nodes:
+    - 9995f539f4f770e2a3fe9e2e615c32a8
+    - 1115f539f4f770e2a3fe9e2e615c32a8
+    - aaa120a9e051d103c164f605615c32a4
+    - bbb340f79047df9bb52fa460615c32a5
+    - ccc8700fe6797ed532e311b0615c32a7
+    - ddd120a9e051d103c164f605615c32a4
+    - eee340f79047df9bb52fa460615c32a5
+    - fff8700fe6797ed532e311b0615c32a7  
+
+node:
+  1115f539f4f770e2a3fe9e2e615c32a8:
+    type: ha_node
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    num_components: 2
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: ha
+      version: 2.0.0-47
+    hostname: ha-node
+    name: ha-node
+    provisioning:              # This is autogenerated/controlled/used by provisioner 
+      phase: deployment
+      status: success
+      version: 2.0.0-576
+    storage_set: storage-set-1
+  9995f539f4f770e2a3fe9e2e615c32a8:
+    type: control_node
+    hostname: control-node
+    name: control-node
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    num_components: 4
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: csm
+      services:
+      - agent
+      version: 2.0.0-21
+    provisioning:
+      phase: deployment
+      status: success
+      version: 2.0.0-576
+    storage_set: storage-set-1
+  eee340f79047df9bb52fa460615c32a5:
+    type: server_node
+    hostname: server-node1
+    name: server-node1
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    num_components: 3
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: hare
+      version: 2.0.0-47
+    - name: s3

```

Comment:
add s3 services section here as it is there in leagy_s3 deployment

Reply:
done